src/main/java/managers/StatusManager.java

```diff
@@ -0,0 +1,15 @@
+package managers;
+
+import java.util.*;
+
+public class StatusManager<E> {
+    private Map<String, Queue> map = new HashMap<>();
+
+    public StatusManager(String[] status) {
+        Arrays.stream(status).forEach(key -> map.put(key, new LinkedList<E>()));
+    }
+
+    public Map<String, Queue> getMap() {
```

Comment:
Why are you returning the whole map? What is the idea of the class if you are returning the whole map?

Reply:
The idea was just generate the queue map from the enums.

But will change that and manage enqueue/dequeue methods  on the  StatusManager class, also the getMap mathod will be removed.